src/modules/asab-config/MicroservicesContainer.js

```diff
@@ -31,9 +34,12 @@ export default (props) => {
 
         const ASABConfigAPI = props.app.axiosCreate('asab_config');
 
-        ASABConfigAPI.get('/microservices').
-            then(res => parseDataIntoArr(res.data)).
-            then(lst => setList(lst));
+        ASABConfigAPI.get('/microservices')
+            .then(res => setList(parseDataIntoArr(res.data)))
+            .catch(e => {
+                console.error(e);
```

Comment:
I think there should be `console.log` instead of `console.error`, because in other project Franta said to me to replace all `console.error(e)` to `console.log` with more information like `console.log("Something went wrong! Failed to ...", e)`